hubblestack/extmods/modules/nebula_osquery.py

```diff
@@ -502,12 +498,8 @@ def check_disk_usage(path=None):
 
         if __salt__['config.get']('splunklogging', False):
             log.debug('Logging disk usage stats to splunk')
-            hubblestack.splunklogging.__grains__ = __grains__
-            hubblestack.splunklogging.__salt__ = __salt__
-            hubblestack.splunklogging.__opts__ = __opts__
-            handler = hubblestack.splunklogging.SplunkHandler()
             stats = {'disk_stats': disk_stats, 'schedule_time': time.time()}
-            handler.emit_data(stats)
+            hubblestack.log.emit_to_splunk(stats, 'INFO', 'hubblestack.osquery_disk_usage')
```

Comment:
In this case I would use a different name for the loggername, this data doesn't specifically report disk usage attributable only to osquery. Rather I think this is path specific.

Reply:
So what would you like me to do? Technically you're correct, but it will only ever be called from osquery code (since it's in that module).

Reply:
That is not explicit or obvious in back end logging system where these messages will end up and be consumed.

Reply:
I disagree. Calling it `hubblestack.disk_usage` or something would be way less clear, since this is only used for osqueryd disk usage.

Reply:
I'm open to alternatives, but you've given me none.